content/applications/websites/ecommerce/managing_products/price_management.rst

```diff
@@ -0,0 +1,240 @@
+:guilabel:`Product Reference Price` under the :guilabel:`Shop - Products`. When enabled, make sure
+an amount is set in the :guilabel:`Base Unit Count` field of the **product template**, as well as in
+the :guilabel:`Sales Price`. The price per unit of measure can be found above the :guilabel:`Add to
+Cart` button on the product page.
+
+.. image:: price_management/price-cost-per-unit.png
+   :align: center
+   :alt: Cost per unit pricing on the product template
+
+.. image:: price_management/price-cost-per-unit-page.png
+   :align: center
+   :alt: Cost per unit pricing on the product page
+
+.. note::
+   Pay attention that having the price per unit may be **mandatory** in some countries.
+
+.. seealso::
+   - :doc:`../../../inventory_and_mrp/inventory/management/products/uom`
+
+Price configuration
+===================
+
+Pricelists
+----------
+
+Pricelists are the main tool to manage prices on your eCommerce. They allow you to define
+website-specific prices - different from the price on the product template - based on **country
+groups**, **currency**, **minimum quantity**, **period**, or **variants**. You can create as many
+pricelists as needed, but it is mandatory to have at least one pricelist available on a website. If
+no custom pricelists are added, Odoo defaults to the **Public Pricelist** for all websites.
+
+Configuration
+~~~~~~~~~~~~~
+
+Pricelists can be found under :menuselection:`Website --> eCommerce --> Pricelists`, but must first
+be activated. For that, head to :menuselection:`Website --> Configuration --> Settings` and scroll
+down to the :guilabel:`Shop - Products` section. There, you can find two options:
+
+- :guilabel:`Multiple prices per product`;
+- :guilabel:`Advanced price rules (discounts, formulas)`.
+
+The **first** option allows you to set different prices per customer *segment*, i.e., registered
+customers, gold customers, regular customers, etc. The **second** option allows you to set *price
+change* rules such as **discounts**, **margins**, **roundings**, etc.
+
+Foreign currency
+~~~~~~~~~~~~~~~~
+
+If you are selling in **multiple currencies** and have pricelists in foreign currencies, customers
+can select their corresponding pricelist anywhere on the :guilabel:`Shop` page from the drop-down
+menu next to the **search bar**.
+
+.. image:: price_management/price-pricelists.png
+   :align: center
+   :alt: Pricelists selection
+
+.. seealso::
+   - :doc:`../../../sales/sales/products_prices/prices/pricing`
+   - :doc:`../../../sales/sales/products_prices/prices/currencies`
+
+Discount and loyalty programs
+=============================
+
+Discount and loyalty
+--------------------
+
+It is possible to create promotion and loyalty programs, which offer different options from
+pricelists. To do so, go to :menuselection:`Website --> eCommerce --> Discount & Loyalty`, and
+:guilabel:`Create` a program from scratch or by using the templates. You can also modify an existing
```

Comment:
```suggestion
:guilabel:`Create` a program from scratch or use one of the templates. You can also modify an existing
```
Note, I don't see the templates displayed below when going to Website --> eCommerce --> Discount & Loyalty

Reply:
Maybe there was a bug, but here's what it looks like FYI 
![template](https://user-images.githubusercontent.com/104567387/217504649-148aa3ab-9ec6-403b-9056-dc5922d94d3d.png)

Reply:
Added a note as well